app.js

```diff
@@ -0,0 +1,147 @@
+// Load Environment Variables from the .env file
+require('dotenv').config();
+
+// Application Dependencies
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const client = require('././lib/client');
```

Comment:
I didn't see either a `lib` directory or `client` file so this won't let you run your server.